### PR TITLE
Added disks info

### PR DESCRIPTION
**Pull request by stellav97**
General info about connected disks, cdrom devices, and partitions of said disks

### CODE REVIEW

**Comment by stellav97** [comment]
ready to be merged!

File: scripts/disks/partitions.sh [hunk 1]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+num=$(lsblk | wc -l)
+echo VALUE BAS number_of_devices $[$num - 1]
+for ((i=2;i<=num;i++))
+do
+	name=$(lsblk | head -n $i | tail -n 1 | tr -s '\t' ' ' | cut -f 1 -d ' ')
```
Comment (MaanooAk) [inline scripts/disks/partitions.sh]: When I run it the names start with `├─`

The output of `./partitions.sh | grep name`
```
VALUE BAS name_1 "sda"
VALUE BAS name_2 "├─sda1"
VALUE BAS name_3 "├─sda2"
VALUE BAS name_4 "└─sda5"
VALUE BAS name_5 "sr0"
```

My lsblk output
```
NAME   MAJ:MIN RM   SIZE RO TYPE MOUNTPOINT
sda      8:0    0 465.8G  0 disk 
├─sda1   8:1    0 461.9G  0 part /
├─sda2   8:2    0     1K  0 part 
└─sda5   8:5    0   3.9G  0 part [SWAP]
sr0     11:0    1  1024M  0 rom  
```

File: scripts/disks/partitions.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	removable=$(lsblk | head -n $i | tail -n 1 | tr -s '\t' ' ' | cut -f 3 -d ' ')
+	size=$(lsblk | head -n $i | tail -n 1 | tr -s '\t' ' ' | cut -f 4 -d ' ')
+	type=$(lsblk | head -n $i | tail -n 1 | tr -s '\t' ' ' | cut -f 6 -d ' ')
+	echo VALUE BAS name_$[$i - 1] \"$name\"
+	echo VALUE BAS major_device_number_$[$i - 1] \"$maj\"
+	echo VALUE BAS minor_device_number_$[$i - 1] \"$min\"
+	echo VALUE ADV mountpoint_$[$i - 1] \"$mountpoint\"
+	echo VALUE ADV read_only_$[$i - 1] \"$read_only\"
+	echo VALUE ADV removable_$[$i - 1] \"$removable\"
+	echo VALUE ADV size_$[$i - 1] \"$size\"
```
Comment (MaanooAk) [inline scripts/disks/partitions.sh]: Also I suggest to make the size BAS and the divice numbers ADV

**Comment by stellav97** [comment]
done , done and done! if there's anything else let me know 😄

**Comment by MaanooAk** [comment]
Im merging. I will add later some grouping commands with #5 to improve the display

**Pull request merged**